Dockerfile

```diff
@@ -23,8 +23,9 @@ RUN /bin/bash -e /security_updates.sh && \
     add-apt-repository ppa:ondrej/nginx -y && \
     apt-get update -yqq && \
     apt-get install -yqq --no-install-recommends \
-        nginx-light \
+        nginx-light=1.22.0-* \
```

Comment:
is this the same version we currently have? or a different one?

Reply:
@ingluisjimenez I think it will pull latest on build.
Version on my vm is `nginx-light | 1.20.1`

Reply:
how does it pull 1.20.1 if it is pinned to `1.22.0-*`?

Reply:
@kangman I mean – currently I have `1.20.1-2+ubuntu20.04.1+deb.sury.org+1` on my machine. 
When I try to build the image with the version
```
    apt-get install -yqq --no-install-recommends \
        nginx-light=1.20.1-* \
        ca-certificates \
        libnginx-mod-brotli=1.20.1-* \
```
i'm getting the error:
```
#4 52.60 E: Version '1.20.1-*' for 'nginx-light' was not found
#4 52.60 E: Version '1.20.1-*' for 'libnginx-mod-brotli' was not found
```
So, when I run `apt-cache madison nginx-light` I see that `1.22.0` version was available.
```
# apt-cache madison nginx-light
nginx-light | 1.22.0-1+ubuntu20.04.1+deb.sury.org+1 | http://ppa.launchpad.net/ondrej/nginx/ubuntu focal/main arm64 Packages
nginx-light | 1.18.0-0ubuntu1.3 | http://ports.ubuntu.com/ubuntu-ports focal-updates/universe arm64 Packages
nginx-light | 1.18.0-0ubuntu1.3 | http://ports.ubuntu.com/ubuntu-ports focal-security/universe arm64 Packages
nginx-light | 1.17.10-0ubuntu1 | http://ports.ubuntu.com/ubuntu-ports focal/universe arm64 Packages```

that's why it's specified there.

@kangman please let me know if you are ok with it.
ps: [changelog](https://launchpad.net/ubuntu/+source/nginx/+changelog)